core/lib/config/src/configs/external_price_api_client.rs

```diff
@@ -14,6 +14,11 @@ pub struct ExternalPriceApiClientConfig {
     /// Forced conversion ratio. Only used with the ForcedPriceClient.
     pub forced_numerator: Option<u64>,
     pub forced_denominator: Option<u64>,
+    /// Forced fluctuation is only used in  ForcedPriceClient. It defines how much percent numerator /
+    /// denominator should fluctuate from their forced values. If it's None or 0, then ForcedPriceClient
+    /// will return the same quote every time it's called. Otherwise, ForcedPriceClient will return
+    /// forced_quote +/- forced_fluctuation % from its values.
+    pub forced_fluctuation: Option<u32>,
```

Comment:
should we have a special config for forcedPriceClient? it's a bit of confusing now

Reply:
Moved to a separate struct